joel/lab2/part1/flake.nix

```diff
@@ -0,0 +1,55 @@
+{
+  description = "fsae onboarding c++ library";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";
+    utils.url = "github:numtide/flake-utils";
+    easy_cmake.url = "github:RCMast3r/easy_cmake";
+  };
+  outputs = { self, nixpkgs, utils, easy_cmake }:
+    let
+      vn_lib_overlay = final: prev: {
+        vn_lib = final.callPackage ./default.nix { };
+      };
+      py_vn_lib_overlay = final: prev: {
+        py_vn_lib = final.callPackage ./python.nix { };
+      };
```

Comment:
Please change names and remove what you don't use

Reply:
done